storage/url/url.go

```diff
@@ -326,6 +326,10 @@ func (u *URL) SetRelative(base *URL) {
 
 // Match reports whether if given key matches with the object.
 func (u *URL) Match(key string) bool {
+	if u.filterRegex == nil {
+		return false
+	}
```

Comment:
Why this was necessary in the scope of this PR?

Reply:
At some time the s5cmd panicked due to nil pointer.

As far as I remember,  I enabled `raw` option with autocompletion URL's ignoring the value of actual raw `flag`. Since the raw URL does not have `filterRegex`, s3.List method panicked here (at URL.Match) method.
Now, I stopped setting 'raw' option to true.